capgains/cli.py

```diff
@@ -29,6 +30,7 @@ def show(transactions_csv, tickers):
 @click.argument('year', type=click.INT)
 @click.option('-t', '--tickers', metavar='TICKERS',
               multiple=True, help="Stocks tickers to filter for")
-def calc(transactions_csv, year, tickers):
-    transactions = TransactionsReader.get_transactions(transactions_csv)
+@click.option('--reverse/--no-reverse', default=False)
```

Comment:
I think it would be useful if you could add a small help string about what this option does.

Also is the `--no-reverse` option needed? A user can just avoid specifying the `--reverse` option to get the default chronological order for the table.